pages/k8s/supported-versions.md

```diff
@@ -0,0 +1,134 @@
+---
+wrapper_template: "kubernetes/docs/base_docs.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "Supported versions"
+  description: The Charmed Kubernetes release cycle and current supported versions.
+keywords: juju, upgrading, track, version
+tags: [operating]
+sidebar: k8smain-sidebar
+permalink: supported-versions.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+Charmed Kubernetes officially supports the most recent three (3) minor releases
+of Kubernetes.
+
+Current Release: **1.17**
+
+Supported releases: **1.17.x, 1.16.x, 1.15.x**
```

Comment:
Do you plan to update this with each release? I'm fine with that if you want to do it. Alternatively we could use some "at the time of this writing" language here. Up to you!

Reply:
yeah, I am planning on writing a script to update all the places which need updating when we do a release, so I think its fine to do this too :)